frontend/src/components/camp/Picasso.vue

```diff
@@ -226,15 +215,10 @@ export default {
       return ''
     },
     entryMouseDown ({ event: entry, timed, nativeEvent }) {
-      if (!entry.tmpEvent && nativeEvent.detail === 2) {
-        // Doubleclick opens activity
-        this.showScheduleEntry(entry)
-      } else if (!entry.tmpEvent && (nativeEvent.button === 1 || nativeEvent.metaKey || nativeEvent.ctrlKey)) {
+      if (!entry.tmpEvent && (nativeEvent.button === 1 || nativeEvent.metaKey || nativeEvent.ctrlKey)) {
         // Click with middle mouse button, or click while holding cmd/ctrl opens new tab
         this.showScheduleEntryInNewTab(entry)
```

Comment:
This triggers a popup blocker in my Firefox 83 on Ubuntu. I think we should open a new issue for fixing that.

Reply:
Yes, I see this as well on FF.

Reply:
#679